VAMobile/src/screens/HomeScreen/HomeScreen.test.tsx

```diff
@@ -52,4 +63,21 @@ context('HomeScreen', () => {
       expect(mockNavigationSpy).toBeCalledWith('Webview', { displayTitle: 'va.gov', url: 'https://www.va.gov/find-locations/', loadingMessage: 'Loading VA location finder...' })
     })
   })
+
+  it('displays prescriptions module when there are active prescriptions', () => {
+    initializeTestInstance(2)
+    expect(screen.getByText('Prescriptions')).toBeTruthy()
+    expect(screen.getByText('(2 active)')).toBeTruthy()
+  })
+
+  it('navigates to prescriptions screen when prescriptions module is tapped', () => {
+    initializeTestInstance(2)
+    fireEvent.press(screen.getByText('Prescriptions'))
+    expect(Linking.openURL).toBeCalledWith('vamobile://prescriptions')
+  })
+
+  it('does not display prescriptions module when there are no active prescriptions', () => {
+    initializeTestInstance(0)
+    expect(screen.queryByText('Prescriptions')).toBeFalsy()
+  })
```

Comment:
These tests will need to be updated when the design is finalized, but I figured I might as well add them since the logic will remain the same